pkgs/development/tools/dbt/default.nix

```diff
@@ -0,0 +1,178 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+, rustPlatform
+}:
+
+let
+  mashumaro = python3Packages.buildPythonPackage rec {
+    pname = "mashumaro";
```

Comment:
I added the latest version in https://github.com/NixOS/nixpkgs/pull/240234, so we could perhaps use it now (relaxing dependency versions if necessary).